plugins/BEdita/Core/src/Model/Entity/ObjectTag.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
```

Comment:
```suggestion
 * Copyright 2022 ChannelWeb Srl, Chialab Srl
```
minor, current year for copyright